packages/react-scripts/scripts/openPlayer.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const chalk = require('react-dev-utils/chalk');
+const fs = require('fs');
+const { execFile } = require('child_process');
+
+async function openPlayer(url) {
+    return new Promise((_, reject) => {
```

Comment:
No need to return a new promise here. The async function will implicitly return a Promise. We can return `Promise.reject()` or throw an error `throw new Error();` 

Does it need to be an async function?

Reply:
Nope, wanted to use Reacts prompt tool, which returns a promise so I started doing it like that then changed it. Forgot I even added the async keyword.

Something interesting - returning Promise.reject(); didn't actually enter the .catch in the start script 🤷 only when I did by returning the whole promise.